counter-strike2config.json

```diff
@@ -0,0 +1,212 @@
+        "DisplayName":"Initial Map",
+        "Category":"CS2 Server Settings",
+        "Description":"Sets the map to start the server with. If \"Custom\" is selected, Custom Initial Map must also be specified",
+        "Keywords":"map",
+        "FieldName":"map",
+        "InputType":"enum",
+        "ParamFieldName":"map",
+        "IncludeInCommandLine":true,
+        "DefaultValue":"de_dust2",
+        "EnumValues":{
+            "de_ancient":"Ancient",
+            "de_anubis":"Anubis",
+            "de_dust2":"Dust II (default)",
+            "de_inferno":"Inferno",
+            "cs_italy":"Italy",
+            "de_mirage":"Mirage",
+            "de_nuke":"Nuke",
+            "cs_office":"Office",
+            "de_overpass":"Overpass",
+            "de_vertigo":"Vertigo",
+            "{{CustomMap}}":"--Custom--"
+        }
+    },
+    {
+        "DisplayName":"Custom Initial Map",
+        "Category":"CS2 Server Settings",
+        "Description":"Sets a custom map to use for the Initial Map, if \"Custom\" is selected",
+        "Keywords":"custom,map",
+        "FieldName":"CustomMap",
+        "InputType":"text",
+        "ParamFieldName":"CustomMap",
+        "DefaultValue":"",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Player Limit",
+        "Category":"CS2 Server Settings",
+        "Description":"Sets the maximum number of players allowed on the server at one time",
+        "Keywords":"max,players,limit,maxplayers",
+        "FieldName":"$MaxUsers",
+        "InputType":"number",
+        "MinValue":"1",
+        "MaxValue":"64",
+        "ParamFieldName":"maxplayers",
+        "DefaultValue":"32",
+        "Placeholder":"32",
+        "Suffix":"players",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Pure Server Mode",
+        "Category":"CS2 Server Settings",
+        "Description":"Sets whether clients are only allowed to use certain files",
+        "Keywords":"pure,server,cheat,hack,integrity,sv_pure",
+        "FieldName":"sv_pure",
+        "InputType":"enum",
+        "ParamFieldName":"sv_pure",
+        "IncludeInCommandLine":true,
+        "DefaultValue":"0",
+        "EnumValues":{
+            "0":"Client files not restricted",
+            "1":"All client files except those listed in pure_server_whitelist.txt must match the server",
+            "2":"All client files must come from Steam"
+        }
+    },
+    {
+        "DisplayName":"Enable Cheats",
+        "Category":"CS2 Server Settings",
+        "Description":"If set, cheat convars will be allowed to be used on the server",
+        "Keywords":"allow,cheat,hack,integrity,sv_cheats",
+        "FieldName":"sv_cheats",
+        "InputType":"checkbox",
+        "ParamFieldName":"sv_cheats",
+        "IncludeInCommandLine":true,
+        "DefaultValue":"false",
+        "EnumValues":{
+            "False":"false",
+            "True":"true"
+        }
+    },
+    {
+        "DisplayName":"Disable Valve Anti-Cheat",
```

Comment:
Not a required change by any means, but when I read this it felt backwards as opposed to a setting that was "Enable Valve Anti-Cheat" that defaults to on.

Reply:
Given the server by default (ie without anything specified) enables VAC, I think it makes more sense to do it this way.